modules/toolbar_tasks/toolbar_tasks.libraries.yml

```
@@ -0,0 +1,5 @@
+toolbar:
+  version: VERSION
+  css:
+    theme:
+      css/toolbar.css: {}
```

```suggestion
      css/toolbar.css: {}
      
```
Add end of file line